neuromation/cli/utils.py

```diff
@@ -436,13 +436,33 @@ def volume_to_verbose_str(volume: Volume) -> str:
     )
 
 
-async def resolve_job(client: Client, id_or_name: str) -> str:
+async def resolve_job(
+    id_or_name_or_uri: str, *, client: Client, default_user: str
+) -> str:
+    if id_or_name_or_uri.startswith("job:"):
+        uri = _normalize_uri(id_or_name_or_uri, username=default_user)
+        id_or_name = uri.path.lstrip("/")
+        assert uri.host
```

Comment:
this assert is redundant (since same check is present in `_normalize_uri`) and is equivalent to `# type: ignore`.